Worker.php

```diff
@@ -2063,7 +2063,7 @@ protected static function writeConnectionsStatisticsToStatusFile()
     public static function checkErrors()
```

Comment:
For maintainability, can we align this with our established patterns in similar modules?